### DISCUSSION TITLE
Plans for using new official Tidal API?

### DISCUSSION THREAD

**Discussion by AlexanderHarrison** [strawberrymusicplayer/strawberry#1388] [General]
[Tidal has released an official API.](https://developer.tidal.com/documentation/api/api-overview) I recall finding my Client ID was a huge pain. Using this API could make it much easier to connect to Tidal. Are there plans to migrate to this?

**Comment by jonaski** [comment]
That API is strictly for metadata, not streaming, but we could use it for album covers, but the closed one we have now actually have higher resolution covers than the open API.

**Reply by AlexanderHarrison** [reply]
Ah, I didn't know it couldn't be used for streaming.

Thanks!
